### PR TITLE
CHE-297 Track not seen outputs in a process

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Signed-off-by: Vitaliy Guliy vguliy@codenvy.com

### CODE REVIEW

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 1]
```diff
@@ -270,6 +271,7 @@ public void addCommandOutput(@NotNull String machineId, @NotNull OutputConsole o
         String outputConsoleTitle = outputConsole.getTitle();
         ProcessTreeNode processTreeNode = getProcessTreeNodeByName(outputConsoleTitle, machineTreeNode);
         if (processTreeNode != null && isCommandStopped(processTreeNode.getId())) {
+            // TODO ???????
```
Comment (azatsarynnyy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: TODO what??????? :smile:

Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: It's not my code below and I'm going to understand what it does.

**Comment by vitaliy-guliy** [comment]
@vparfonov ?

File: core/commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/tree/Tree.java [hunk 2]
```diff
@@ -443,19 +443,22 @@ protected void onTreeNodeBodyChildEvent(Event evt, Element treeNodeBody) {
...
-                        getDelegate().onDragDropEvent((MouseEvent)event);
-                    }
-                }
-            };
-            getElement().addEventListener(Event.DROP, dragDropEventListener, false);
-            getElement().addEventListener(Event.DRAGOVER, dragDropEventListener, false);
-            getElement().addEventListener(Event.DRAGENTER, dragDropEventListener, false);
-            getElement().addEventListener(Event.DRAGLEAVE, dragDropEventListener, false);
-            getElement().addEventListener(Event.DRAGSTART, dragDropEventListener, false);
+//            Temporary disable drag & drop on the tree
```
Comment (vparfonov) [inline core/commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/tree/Tree.java]: do we support drag & drop ?

Comment (vitaliy-guliy) [inline core/commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/tree/Tree.java]: Sure not. But it was turned on before.

Comment (vparfonov) [inline core/commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/ui/tree/Tree.java]: So remove commented code

**Comment by vparfonov** [comment]
ok

**Pull request merged**